core-plugin/src/com/google/gct/idea/debugger/ui/CloudDebugHistoricalSnapshots.java

```diff
@@ -262,9 +262,6 @@ private void onBreakpointsChanged() {
 
     if (breakpointList != null) {
       for (int i = 0; i < breakpointList.size(); i++) {
-        if (breakpointList.get(i).getFinalTime() == null) { // todo: what does this mean?
```

Comment:
This was here because the pending snapshots didn't have times. As a side effect of my previous PR, now they do. 

There is an alternate approach I could take here. I could sue a separate code path for pending snapshots that deos not add a time. However that would increase the cyclomatic complexity of the code rather than decrease it as this fix does. epicycles on epicycles.